docs/operator/cn/03-10-operations.md

```diff
@@ -0,0 +1,22 @@
+# {Component/Service Name} - Operations
+
+<!--Please create only one single document with multiple sections for your Operations documentation. Our goal is to display all Operations content for a service on a single page so that the operators can quickly search for information using the search function.-->
+
+<!--Recommended sections:
```

Comment:
One more section came to my mind we need to add here: Metrics

Each module and component needs to document the followings:
 - What metrics endpoints are there, and in what format they are producing metrics (prometheus is expected here generally)
 - What metrics are presented on each endpoint, and each and every of these need sufficient documentation for the reader to understand what kind of internal mechanics it is representing
 - Suggested way to determine warnings and critical levels for metrics

Separately for alerting, if some important conditions are represented by a combination of metrics, then these should also be listed as suggested alerts, along with threshold mechanics.